src/libraries/Microsoft.PowerFx.Core/IR/IRTranslator.cs

```diff
@@ -254,7 +261,17 @@ public override IntermediateNode Visit(TexlCallNode node, IRTranslatorContext co
                 for (var i = 0; i < carg; ++i)
                 {
                     var arg = node.Args.Children[i];
-                    if (func.IsLazyEvalParam(i))
+
+                    var supportIdentifiers = _features.HasFlag(Features.SupportIdentifiers);
+                    if (supportIdentifiers && func.IsIdentifier(i))
+                    {
+                        var identifierNode = arg.AsIdentifierNode();
+                        Contracts.Assert(identifierNode != null);
+
+                        // Transform the identifier node as a string literal
+                        args.Add(new TextLiteralNode(context.GetIRContext(arg, DType.String), identifierNode.GetName()));
```

Comment:
Actually, we'll need "ShowColumns()" for this. 
Since that arg1 references to an existing field on arg0 (which is a common pattern). 
Whereas AddColumns is special since it's creating a new column name (so it doesn't appear on arg 0).

Reply:
see [https://github.com/microsoft/Power-Fx/blob/main/src/tests/Microsoft.PowerFx.Core.Tests/DisplayNameTests.cs](https://github.com/microsoft/Power-Fx/blob/main/src/tests/Microsoft.PowerFx.Core.Tests/DisplayNameTests.cs)

Reply:
I'd need your help to understand how to define a display name

Reply:
This also applies to DropColumns, those are referencing fields in the type and may have display names.

Reply:
see what we do for first names to ensure it's resolved correctly:

``` 
                var nodeName = context.Binding.TryGetReplacedIdentName(node.Ident, out var newIdent) ? new DName(newIdent) : node.Ident.Name;
```

Reply:
I've done what you requested but can't say I've understood anything.
I've too many unanswered questions.
"see what we do for first names" -> if that's in Visit(FirstNameNode node, IRTranslatorContext context), I don't understand that code, don't know what LambdaFullRecord, LambdaField, OptionSet, PowerFxResolvedObject "kinds" are + What output is it generating?
"ensure it's resolved correctly" -> Am with you but I wish I'd understand what we're doing here in this method Visit(TexlCallNode node...) -  Why is the method called "Visit"? What is an "IntermediateNode" (apart from an IRContext?)? What is IRTranslatorContext? Why do we call MaybeInjectCoercion at the end?